src/components/no-periods.cjsx

```diff
@@ -0,0 +1,21 @@
+_ = require 'underscore'
+React = require 'react'
+BS = require 'react-bootstrap'
+
+NO_PERIODS_TEXT = 'Please add at least one period to the course.'
+
+NoPeriods = React.createClass
+
+  propTypes:
+    noPanel:   React.PropTypes.bool
+
+  render: ->
+    text = <span className='-no-periods-text'>{NO_PERIODS_TEXT}</span>
```

Comment:
Should this also have a link to the "Roster"? I'd probably still merge this in as-is and then have a separate PR to fix up the boilerplate text.

/cc @jason-holmes or @Fredasaurus